docker-compose.test.watch.yaml

```diff
@@ -7,7 +7,7 @@ services:
     volumes:
       - ./app:/home/node/app
       - ./test:/home/node/test
-      - ./jest.config.js:/home/node/jest.config.js
+      - ./jest.config.cjs:/home/node/jest.config.cjs
```

Comment:
im surprised we dont need to put the babel config in here too - how is it knowing how to parse the JS when it runs in docker?

Reply:
<img width="250" alt="Screenshot 2025-02-13 at 15 50 38" src="https://github.com/user-attachments/assets/fe703e44-aaaf-4dc4-839f-938207ac286d" />
Because of this from the Dockerfile basically. All files end up inside the image